### PR TITLE
WIP: Aplite love ❤️

### PR DESCRIPTION
**Pull request by tertty**
Opening a PR to create an initial/ongoing fix for issue #56. 

Snowy has been getting a lot of attention, and Aplite is getting jealous. I hope to maintain the same clean, consistent look currently in Snowy and bring it over to Aplite as we continue to make changes to the system.

### Menus before
image

### Menus now
image

### Behind the scene changes
All original Pebble [Macros](https://developer.get-rpws.com/guides/best-practices/building-for-every-pebble/index.html#available-defines-and-macros) have been added to rwatch/pebble_defines.h! This will increase app/watchface compatibility that relied on these Marcos rather than using #define (This will only work on current RebbleOS supported models).

### Proposed future changes

- Create dynamic B/W color shifting app icons in system menu (When the black bar goes over black icons, they disappear currently.)

- Determine what code can be converted to Macros rather than using #define (systemapp.c converted in this PR as an idea of what it would look like.)

- R̶e̶n̶a̶m̶e̶ ̶M̶a̶c̶r̶o̶s̶ ̶t̶o̶ ̶M̶a̶r̶c̶o̶

- Any more ideas?

### CODE REVIEW

**Comment by jwise** [comment]
Amazing!!! I haven't looked at the code yet but the results are great.  Only one question ... ¿quién es Marcos?

File: Apps/System/systemapp.c [hunk 1]
```diff
@@ -184,14 +182,9 @@ static void about_window_load(Window *window)
...
-    #ifdef PBL_BW
-         //TODO: Get black and white rocket bitmap for Classic
-        logo_bitmap = gbitmap_create_with_resource(RESOURCE_ID_REBBLE_LOGO_BW);
-        rocket_bitmap = gbitmap_create_with_resource(RESOURCE_ID_TO_MOON_BW);       
-    #else
-        logo_bitmap = gbitmap_create_with_resource(RESOURCE_ID_REBBLE_LOGO_DARK);
-        rocket_bitmap = gbitmap_create_with_resource(RESOURCE_ID_TO_MOON);
-    #endif
+    //TODO: Get black and white rocket bitmap for Classic
+    logo_bitmap = gbitmap_create_with_resource(PBL_IF_BW_ELSE(RESOURCE_ID_REBBLE_LOGO_BW, RESOURCE_ID_REBBLE_LOGO_DARK));
```
Comment (jwise) [inline Apps/System/systemapp.c]: Maybe another idea is to just give it the same resource name in the .json file on both platforms?

File: Apps/System/systemapp.c [hunk 2]
```diff
@@ -120,12 +120,10 @@ static void exit_to_watchface(struct Menu *menu, void *context)
...
     Layer *window_layer = window_get_root_layer(s_main_window);
-#ifdef PBL_RECT
-    s_menu = menu_create(GRect(0, 16, DISPLAY_COLS, DISPLAY_ROWS - 16));
-#else
-    // Let the menu draw behind the statusbar so it is perfectly centered
-    s_menu = menu_create(GRect(0, 0, DISPLAY_COLS, DISPLAY_ROWS));
-#endif
+
+    // For round, let the menu draw behind the statusbar so it is perfectly centered
+    s_menu = menu_create(PBL_IF_RECT_ELSE(GRect(0, 16, DISPLAY_COLS, DISPLAY_ROWS - 16), GRect(0, 0, DISPLAY_COLS, DISPLAY_ROWS)));
```
Comment (jwise) [inline Apps/System/systemapp.c]: Maybe on two different lines, aligned with spaces, to better visually indicate the difference?  i.e., 

```c
    s_menu = menu_create(PBL_IF_RECT_ELSE(GRect(0, 16, DISPLAY_COLS, DISPLAY_ROWS - 16), // rect
                                          GRect(0, 0, DISPLAY_COLS, DISPLAY_ROWS)        // round
                                         ));
```

File: Apps/System/menu.c [hunk 3]
```diff
@@ -118,7 +118,7 @@ Menu* menu_create(GRect frame)
     Menu *menu = (Menu *) app_calloc(1, sizeof(Menu));
     menu->items = menu_items_create(0);
     menu->layer = menu_layer_create(frame);
-    menu_layer_set_highlight_colors(menu->layer, GColorRed, GColorWhite);
+    menu_layer_set_highlight_colors(menu->layer, PBL_IF_COLOR_ELSE(GColorRed,GColorBlack), GColorWhite);
```
Comment (ginge) [inline Apps/System/menu.c]: (GColorRed,GColorBlack)  space please

**Comment by tertty** [comment]
Updated branch with suggestions made in this PR so far. I didn't touch the resource ID's yet as I think it would be best to start reorganizing that as we start adding more icons and have a better game plan for resources.

**Comment by tertty** [comment]
PR is super out of date/need to be completely reworked due to advancements in project. Closing out 👀

**Pull request closed without merging**